airflow/models/taskinstance.py

```diff
@@ -927,6 +929,10 @@ def signal_handler(signum, frame):
                 start_time = time.time()
 
                 self.render_templates(context=context)
+                if STORE_SERIALIZED_DAGS:
```

Comment:
Quite generic question, maybe even an open discussion here.
STORE_SERIALIZED_DAGS comes from settings.py and is loaded when application (like webserver or scheduler) starts.
If we use such variables this property stops being dynamic.
But with scheduler reloading dags every now and then, it could be.

Do we want to enforce that changing core.store_serialized_dags only takes effect if you restart?
It is not crucial that it is dynamic but if it can be dynamic why should not it be?

Reply:
constant vs reading from `conf.getboolean` everytime wouldn't make a difference, as the `airflow.config` is loaded once at main-process start up, and the parsing sub-process forks so keeps the same loaded config -- i.e. it doesn't re-read the config file.